styles.css

```diff
@@ -0,0 +1,319 @@
+  /* max-width: 50px; */
+  max-height: 45px;
+  filter: invert(1);
+  opacity: 0.5;
+}
+
+section .container .contactinfo .sci li:hover a img {
+  opacity: 1;
+}
+
+section .container .contact-form {
+  position: absolute;
+  padding: 70px 50px;
+  background: #fff;
+  width: calc(100% - 250px);
+  margin-left: 150px;
+  padding-left: 350px;
+  height: 100%;
+  box-shadow: 0 50px 50px rgba(0, 0, 0, 0.5);
+}
+
+section .container .contact-form h2 {
+  color: #0f3959;
+  font-size: 24px;
+  font-weight: 500;
+}
+
+section .container .contact-form .formbox {
+  padding: 20px;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+section .container .contact-form .formbox .inputbox {
+  position: relative;
+  padding: 0 0 35px 0;
+}
+
+section .container .contact-form .formbox .inputbox.w50 {
+  width: 47%;
+}
+
+section .container .contact-form .formbox .inputbox.w100 {
+  width: 100%;
+}
+
+section .container .contact-form .formbox .inputbox input,
+section .container .contact-form .formbox .inputbox textarea {
+  width: 100% !important;
+  color: #333;
+  padding: 5px;
+
+  font-size: 18px;
+  /* This will remove the existing borders */
+  border: none;
+  /* This will create only bottom border like underline effect */
+  border-bottom: 1px solid #777;
+  font-weight: 300px;
+  /* This is used so no line is visible when selecting the input */
+  outline: none;
+  /* this will not allow to resize the text area */
+  resize: none;
+}
+
+section .container .contact-form .formbox .inputbox textarea {
+  min-height: 120px;
+}
+
+section .container .contact-form .formbox .inputbox span {
+  position: absolute;
+  left: 0;
+  padding: 5px 0;
+  font-size: 18px;
+  font-weight: 300px;
+  color: #333;
+  transition: 0.5s;
+  /* the span name wasn't clickable, after this it became normal */
+  pointer-events: none;
+}
+
+/* :focus makes the element move on clicking, 
+the translateY(20px); makes it goes in the X direction by 20px on clicking
+:valid makes the focus not retract when the text is written, or else it used
+to overlap the written text after defocusing.
+*/
+section .container .contact-form .formbox .inputbox input:focus ~ span,
+section .container .contact-form .formbox .inputbox textarea:focus ~ span {
+  transform: translateY(-22px);
+  font-size: 12px;
+  font-weight: 400;
+  letter-spacing: 1px;
+  color: firebrick;
+}
+section .container .contact-form .formbox .inputbox input:valid ~ span,
+section .container .contact-form .formbox .inputbox textarea:valid ~ span {
+  transform: translateY(-22px);
+  font-size: 12px;
+  font-weight: 400;
+  letter-spacing: 1px;
+  color: firebrick;
+}
+
+section .container .contact-form .formbox .inputbox input[type="submit"] {
+  position: relative;
+  cursor: pointer;
+  background: #0f3f59;
+  color: #fff;
+  border: none;
+  max-width: 150px;
+  padding: 12px;
+}
+
+section .container .contact-form .formbox .inputbox input[type="submit"]:hover {
```

Comment:
In CSS selectors no need to select it from its parent, just select last 2 or 3 selectors so it can be specific than other selectors.

Reply:
Before, I used to select by 2-3 selectors only, But in the Youtube Video he used this way, so I thought maybe this is the correct way to select it to make sure u dont select anything else (clashing).